internal/adapter/in/controller/user_controller_test.go

```
@@ -66,12 +65,13 @@ var _ = Describe("User Controller", func() {
 					Name: userNamespacedName.Namespace,
 				},
 			}
-			k8sClient.Create(ctx, ns)
+			err := k8sClient.Create(ctx, ns)
+			Expect(err).NotTo(HaveOccurred())
```

The tests are failing due to that the namespace already exists.
Ignore any errors during namespace creation.